test/server/cards/02_SHD/upgrades/BrutalTraditions.spec.ts

```diff
@@ -0,0 +1,48 @@
+describe('Brutal Traditions', function() {
+    integration(function(contextRef) {
+        describe('Brutal Tradition\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'vigilant-honor-guards', upgrades: ['brutal-traditions'] }],
+                    },
+                    player2: {
+                        groundArena: ['wampa'],
+                        hand: ['confiscate']
+                    }
+                });
+            });
+
+            it('should be able to play it from the discard pile when an opponent\'s unit is defeated.', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.vigilantHonorGuards);
+                context.player1.clickCard(context.wampa);
+
+                expect(context.player2).toBeActivePlayer();
+                expect(context.wampa).toBeInLocation('discard');
+
+                context.player2.clickCard(context.confiscate);
+
+                expect(context.brutalTraditions).toBeInLocation('discard');
+                expect(context.player1).toBeActivePlayer();
+                expect(context.player1).toBeAbleToSelect(context.brutalTraditions);
+
+                context.player1.clickCard(context.brutalTraditions);
+                expect(context.vigilantHonorGuards.upgrades).toEqual([context.brutalTraditions]);
+                expect(context.vigilantHonorGuards.getPower()).toBe(5);
+                expect(context.vigilantHonorGuards.getHp()).toBe(8);
+            });
+
+            it('should not be able to play it from the discard pile.', function () {
```

Comment:
Actually we can test two things at once here - if we merge this into the above test but do it after `moveToNextActionPhase()`, we confirm that the condition is over when the phase changes and that the card is not playable when the condition is not met

Reply:
yessir! Made them into one!